nucypher/characters/lawful.py

```diff
@@ -163,6 +165,38 @@ def get_policy_pubkey_from_label(self, label: bytes) -> UmbralPublicKey:
         policy_pubkey = alice_delegating_power.get_pubkey_from_label(label)
         return policy_pubkey
 
+    def revoke(self, policy):
+        """
+        Parses the treasure map and revokes arrangements in it.
+        If any arrangements can't be revoked, then the node_id and
+        arrangement_id are added to a list and returned. If a node returns a
+        404 status code, then it will add a tuple of the Revocation and `None`.
+        """
+        # Sign the revocations in the RevocationKit in preparation for
+        # sending to Ursula.
+        policy.revocation_kit.sign_revocations(self.stamp)
+
+        try:
+            # Wait for a revocation threshold of nodes to be known ((n - m) + 1)
+            revocation_threshold = ((policy.n - policy.treasure_map.m) + 1)
+            self.block_until_specific_nodes_are_known(
+                    policy.revocation_kit.revokable_addresses,
+                    allow_missing=revocation_threshold)
```

Comment:
Should this not be `n - revocation_threshold`?

Reply:
Yeah, seems to me that it needs to be `n - revocation_threshold`.